backend/app.js

```diff
@@ -0,0 +1,36 @@
+const express = require('express');
+const cors = require('cors'); // Import the cors package
+const adminRoutes = require('./routes/adminRoutes');
+const authRoutes = require('./routes/authRoutes');
+const bookingRoutes = require('./routes/bookingRoutes');
+const passengerRoutes = require('./routes/passengerRoutes');
+const tripRoutes = require('./routes/tripRoutes');
+
+const app = express();
+const port = 3000;
+
+// Enable CORS for all routes
+app.use(cors());  // This allows all origins to access your API
+
```

Comment:
We can add here a middleware for logging requests

```javascript
app.use(morgan('dev'));
```

Same as before, if you want to use it, include it in the header:

```javascript
const morgan = require('morgan');
```